hello_world/Cargo.toml

```diff
@@ -1,5 +1,5 @@
 [package]
-name = "hello-world"
+name = "hello_world"
 version = "0.1.0"
 authors = ["Ameya Palande <2ameya@gmail.com>"]
```

Comment:
Feel free to add `"Anish Karandikar <anishkny@gmail.com>"` 😄

Reply:
Feel free to add it in your next PR.